classes/03class/exercises/c03-aws03/nimmi89/README.md

```diff
@@ -0,0 +1,139 @@
+# c03-aws03
+
+## Command Execution Output
+- Command used to create the Route53 zone:
+```
+>>Command to create Private hosted zone
+aws route53 create-hosted-zone\
+    --name pvt-zone.da\
+    --vpc VPCRegion=ap-southeast-2,VPCId=vpc-081afcee350babdf5\ --caller-reference 12 JUL 2020\
+    --hosted-zone-config Comment="DA-PVT ZONE",PrivateZone=true
+
+>The command returns:
+{
+    "Location": "https://route53.amazonaws.com/2013-04-01/hostedzone/Z028320826FOJTFR4SOVJ",
+    "HostedZone": {
+        "Id": "/hostedzone/Z028320826FOJTFR4SOVJ",
+        "Name": "pvt-zone.da.",
+        "CallerReference": "12-07-2020",
+        "Config": {
+            "Comment": "DA-PVT ZONE",
+            "PrivateZone": true
+        },
+        "ResourceRecordSetCount": 2
+    },
+    "ChangeInfo": {
+        "Id": "/change/C0244904UBVXPVHK2EY2",
+        "Status": "PENDING",
+        "SubmittedAt": "2020-07-12T00:46:27.743000+00:00"
+    },
+    "VPC": {
+        "VPCRegion": "ap-southeast-2",
+        "VPCId": "vpc-081afcee350babdf5"
+    }
+}
+```
+
+- Command used to add any new records to the zone 
+```
+aws route53 change-resource-record-sets --hosted-zone-id /hostedzone/Z028320826FOJTFR4SOVJ  --change-batch file://da-record.json
+
+>FILE: da-record.json
+{
+            "Comment": "CREATE a record ",
+            "Changes": [{
+            "Action": "CREATE",
+                        "ResourceRecordSet": {
+                                    "Name": "myalb.pvt-zone.da",
+                                    "Type": "CNAME",
+                                    "TTL": 300,
+                                 "ResourceRecords": [{ "Value": "my-load-balancer-203056255.ap-southeast-2.elb.amazonaws.com"}]
+}}]
+}
+
+>The command returns:
+{
+    "ChangeInfo": {
+        "Id": "/change/C07888573JBCET5FJ32M3",
+        "Status": "PENDING",
+        "SubmittedAt": "2020-07-12T00:53:53.587000+00:00",
+        "Comment": "CREATE a record "
+    }
+}
+
+>> Command to check the change status
+aws route53  get-change --id /change/C07888573JBCET5FJ32M3
+{
+    "ChangeInfo": {
+        "Id": "/change/C012091729908YN2ESM0K",
+        "Status": "INSYNC",
+        "SubmittedAt": "2020-07-12T00:22:34.386000+00:00",
+        "Comment": "CREATE a record "
+    }
+}
+
+>>SSH into instance to check the working
+{
+    "ChangeInfo": {
+        "Id": "/change/C07888573JBCET5FJ32M3",
+        "Status": "INSYNC",
+        "SubmittedAt": "2020-07-12T00:53:53.587000+00:00",
+        "Comment": "CREATE a record "
+    }
+}
```

Comment:
I think this is duplicated with lines L66-73

Reply:
Oh!yes..by misatke! have fixed!